osu.Game.Rulesets.Diva/DivaRuleset.cs

```diff
@@ -139,7 +139,7 @@ public override IEnumerable<Mod> GetModsFor(ModType type)
             new KeyBinding(InputKey.W, DivaAction.Triangle),
             new KeyBinding(InputKey.S, DivaAction.Cross),
             new KeyBinding(InputKey.D, DivaAction.Circle),
-            
+
```

Comment:
I struggle to see the change on mobile, what is that empty line?

Reply:
line with spaces to empty line